README.md

```diff
@@ -33,14 +30,15 @@ the project.
     - [Experimental Joint Matrix Support](#jm_support)
   - [Requirements](#requirements)
   - [Setup](#setup)
-    - [Compile with ComputeCpp](#compile-with-computecpp)
     - [Compile with DPC++](#compile-with-dpc)
     - [Compile with hipSYCL](#compile-with-hipsycl)
```

Comment:
We should update all reference to hipSYCL
```suggestion
    - [Compile with AdaptiveCpp](#compile-with-adaptivecpp)
```

Reply:
Thanks for the comments Nicolo. I actually avoided updating the hipSYCL->AdaptiveCpp naming here as it would require updating the cmake configurations as well to match the naming convention. I've put these changes in the overall update of the hipSYCL backend instead.